api/handlers.go

```diff
@@ -1229,6 +1261,14 @@ func handleContentUnits(db *sql.DB, r ContentUnitsRequest) (*ContentUnitsRespons
 	if ex != nil {
 		return nil, ex
 	}
+	// tags
+	for idx, unit := range units {
+		cu := cus[idx]
+		cu.Tags = make([]string, len(unit.R.Tags))
+		for i, x := range unit.R.Tags {
+			cu.Tags[i] = x.Pattern.String
```

Comment:
Do we really need Pattern ? Pattern is for naming of physical files. I don't think it's used in this project. I think you need tag_i18n ...
Moreover, pattern is a nullable field so real guarantee that we'll have something in that field.
In addition, you change semantics of `cu.Tags` because you put different data in it.